### DISCUSSION TITLE
hold hotkeys to increase or decrease window size

### DISCUSSION THREAD

**Discussion by yihou** [rxhanson/Rectangle#1243] [Q&A]
Is there a way to hold the hotkeys to continuously increase or decrease the size of the window?

**Comment by rxhanson** [comment]
At the moment there is not. I don't mind merging a good pull request that adds this functionality though.

**Comment by luizbrotherhood** [comment]
I've been wishing for this feature for a long time! Could this be done with the "Shortcuts" app, apple script, or other third-party software?
